layouts/partials/_elements/grid.html

```diff
@@ -0,0 +1,26 @@
+{{- $outline := default "false" .outline -}}
```

Comment:
Is it possible to add a comment here, without generating blank lines in the output, which documents the input parameters expected by the element?